script_create_branches.py

```diff
@@ -1,40 +1,119 @@
 import os
```

Comment:
On general comment - you seem to have some inconsistency in formatting. To fix that please install `black` library. Then you can run it: 
```
# to check if files have proper formatting type in console:
black --check /path/to/your/files/root

# to actually fix the files:
black  /path/to/your/files/root
```

Reply:
done